scripts/S3_FSCK/s3_fsck_p1.py

```diff
@@ -39,37 +39,65 @@
 
 
 files = "%s://%s/%s/listkeys.csv" % (PROTOCOL, PATH, RING)
+# reading without a header,
+# columns _c0, _c1, _c2, _c3 are the default column names of
+# columns   1,   2,   3,   4 for the csv
+# input structure: (RING key, main chunk, disk, flag)
+# e.g. 555555A4948FAA554034E155555555A61470C07A,8000004F3F3A54FFEADF8C00000000511470C070,g1disk1,0
+# Required Fields:
+#   - _c1 (main chunk)
+#   - _c3 (FLAG)
 df = spark.read.format("csv").option("header", "false").option("inferSchema", "true").option("delimiter", ",").load(files)
```

Comment:
This is another spot we can inject valid headers prior to later commands, making them a bit simpler to comprehend:

```suggestion
df = spark.read.format("csv").option("header", "false").option("inferSchema", "true").option("delimiter", ",").load(files)\
df = df.withColumnRenamed("_c0", "ringkey").withColumnRenamed("_c1", "mainchunk").withColumnRenamed("_c2", "disk").withColumnRenamed("_c3", "flag")
```

in this example I name the **_c0** (ring chunk keys) as ringkey, instead of the naming **_c1**, the main chunk, as **ringkey**. I think this could potentially reduce confusion if we decide to be very specific and use explicit terms for each data type
* ringkey (or ring_key)
* digkey (or dig_key)
* disk
* flag
* inputkey (or input_key)
* mainchunk (or main_chunk)

I like the underscore versions for better readability.
*

Reply:
whatever's easier to read is fine by me